### ISSUE TITLE
DeprecationWarning: use options instead of chrome_options...

### ISSUE THREAD

**Issue by giannisilv** [loblab/noip-renew#22]
I have this error:

pi@RaspGarden:~/rpi-noip-renew $ ./noip-renew.sh
./noip-renew.py:39: DeprecationWarning: use options instead of chrome_options
  self.browser = webdriver.Chrome(chrome_options=options)
2020/06/03 23:07:36 [giannisilv@libero.it] - Debug level: 2
2020/06/03 23:07:36 [giannisilv@libero.it] - Open https://www.noip.com/login...
2020/06/03 23:09:32 [giannisilv@libero.it] - Message: timeout: Timed out receiving message from renderer: 1,075
  (Session info: headless chrome=78.0.3904.108)

Traceback (most recent call last):
  File "./noip-renew.py", line 109, in run
    self.login(username, password)
  File "./noip-renew.py", line 53, in login
    self.browser.save_screenshot("debug1.png")
  File "/usr/local/lib/python3.7/dist-packages/selenium/webdriver/remote/webdriver.py", line 1055, in save_screenshot
    return self.get_screenshot_as_file(filename)
  File "/usr/local/lib/python3.7/dist-packages/selenium/webdriver/remote/webdriver.py", line 1032, in get_screenshot_as_file
    png = self.get_screenshot_as_png()
  File "/usr/local/lib/python3.7/dist-packages/selenium/webdriver/remote/webdriver.py", line 1064, in get_screenshot_as_png
    return base64.b64decode(self.get_screenshot_as_base64().encode('ascii'))
  File "/usr/local/lib/python3.7/dist-packages/selenium/webdriver/remote/webdriver.py", line 1074, in get_screenshot_as_base64
    return self.execute(Command.SCREENSHOT)['value']
  File "/usr/local/lib/python3.7/dist-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/usr/local/lib/python3.7/dist-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 1,075
  (Session info: headless chrome=78.0.3904.108)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "./noip-renew.py", line 138, in <module>
    sys.exit(main())
  File "./noip-renew.py", line 135, in main
    return robot.run(username, password, num_hosts)
  File "./noip-renew.py", line 114, in run
    self.browser.save_screenshot("exception.png")
  File "/usr/local/lib/python3.7/dist-packages/selenium/webdriver/remote/webdriver.py", line 1055, in save_screenshot
    return self.get_screenshot_as_file(filename)
  File "/usr/local/lib/python3.7/dist-packages/selenium/webdriver/remote/webdriver.py", line 1032, in get_screenshot_as_file
    png = self.get_screenshot_as_png()
  File "/usr/local/lib/python3.7/dist-packages/selenium/webdriver/remote/webdriver.py", line 1064, in get_screenshot_as_png
    return base64.b64decode(self.get_screenshot_as_base64().encode('ascii'))
  File "/usr/local/lib/python3.7/dist-packages/selenium/webdriver/remote/webdriver.py", line 1074, in get_screenshot_as_base64
    return self.execute(Command.SCREENSHOT)['value']
  File "/usr/local/lib/python3.7/dist-packages/selenium/webdriver/remote/webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "/usr/local/lib/python3.7/dist-packages/selenium/webdriver/remote/errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.TimeoutException: Message: timeout: Timed out receiving message from renderer: 6,173
  (Session info: headless chrome=78.0.3904.108)

I have a Raspberry 2, Raspbian 10 lite (Buster).

Can you help me???

**Comment by IDemixI** [comment]
Hi @giannisilv,

What version of this script are you using? I would remove the script completely and re-install it. The line erroring above reads as: `self.browser = webdriver.Chrome(chrome_options=options)`

While in the latest version of this script, the line is `browser = webdriver.Chrome(options=options)
`

I'm using a Raspberry Pi 2 B+ with Raspbian 10 Buster as well and it works great for me.

So, to summarise,

- Remove the script completely.
- Use the following command to download the script again: `git clone https://github.com/loblab/noip-renew.git`
- Run setup.sh
- Enter your noip.com account information
- To run the script after installation, simply type `noip-renew-USERNAME` (typing noip-renew- followed by tab will auto-fill this for you.

Any issues, let me know.

**Comment by giannisilv** [comment]
Thanks a lot. I change the line in options=options
and now it works well!!!!
